pkg/controller/controlplane/valuesprovider_test.go

```diff
@@ -145,24 +144,6 @@ var _ = Describe("ValuesProvider", func() {
 			},
 		}
 
-		clusterK8sLessThan118 = &extensionscontroller.Cluster{
-			ObjectMeta: metav1.ObjectMeta{
-				Annotations: map[string]string{
-					"generic-token-kubeconfig.secret.gardener.cloud/name": genericTokenKubeconfigSecretName,
-				},
-			},
-			Shoot: &gardencorev1beta1.Shoot{
-				Spec: gardencorev1beta1.ShootSpec{
-					Networking: gardencorev1beta1.Networking{
-						Pods: &cidr,
-					},
-					Kubernetes: gardencorev1beta1.Kubernetes{
-						Version: "1.17.1",
-					},
-				},
-			},
-		}
-
 		clusterK8sAtLeast118 = &extensionscontroller.Cluster{
```

Comment:
Do we still need it? Can we make it `clusterK8sAtLeast120` and set the k8s version to 1.20.x?